python/feathub/table/table_descriptor.py

```diff
@@ -54,6 +54,7 @@ def __init__(
         keys: Optional[List[str]],
         timestamp_field: Optional[str],
         timestamp_format: str = "epoch",
+        attributes: Optional[Dict[str, str]] = None,
```

Comment:
Would it be better to reserve some keywords that might be applicable to all tables? For example, reserve all keywords that start and end with double underscores, so that Feathub may define common attributes like `__DESCRIPTION__`.

Reply:
I think Feathub doesn't need to reserve keywords in the attribute and users can always add the description to the attributes, e.g. "description": "...". What do you think?

Reply:
We need to predefine field names for common information such as `description`.  This is necessary for UI to programmatically analyze and display that common information.

Maybe add `description: str` to the `Feature` constructor?